### DISCUSSION TITLE
Browser sent to MSN on install

### DISCUSSION THREAD

**Discussion by Phrede** [arendst/Tasmota#20113] [Q&A]
Not that big of a deal, but I'm curious...

There are times when a Tasmota device looses it's wifi connection, caused by me,  and after I connect via the AP SSID my browser is sent to some MSN website.  I just over write 192.168.4.1 and move on.   It's happened many times.

**Comment by barbudor** [comment]
This is a standard behavior of browser. When you connect to an AP that has no direct access to the Internet it call a website (MSN on Windows, probably something else on other OS) and the router of the AP is supposed to catch that to display it's connection page.
That's what happens when you connect your laptop to a public Hotspot.

In your case, I assume that your computer was still connected to the internet by a wired connection. So the computer found an easier path to reach MSN than though Tasmota.
Same with a mobile that has a broadband data. 
Not Tasmota fault, nothing we could do there.

Next time, try to disconnect from your wired network/mobile data to confirm that behavior.
